src/cli/index.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable no-console */
+import { execSync } from 'node:child_process'
+import fs from 'node:fs'
+import path from 'node:path'
+import process from 'node:process'
+import c from 'kleur'
+
+export async function cli() {
+  const cwd = process.cwd()
+
+  console.log(c.dim(`\nSetup eslint config...\n`))
+
+  // Update package.json
```

Comment:
We might have a few prompts for each step.

Reply:
Im not quite sure what exactly is meant, and Im also not sure I can envision a completely correct `promts` usage scenario on your behalf, so as best I could. 

By the way, what do you think about splitting the functionality into separate parts, e.g.
```ts
export async function cli() {
  console.log(c.dim(`\nSetup eslint config ...\n`))

  const pkg = await updatePackageJson()
  await updateEslintFiles(pkg)
  await updateSettingsJson()

  console.log(c.green('\nSuccessful. Now you can update the dependencies and run'), c.dim('eslint . --fix\n')))
}
```
Or is it better to leave it as it is?